Sources/StreamChat/StateLayer/Chat.swift

```diff
@@ -170,6 +182,70 @@ public final class Chat {
         try await channelUpdater.removeMembers(currentUserId: currentUserId, cid: cid, userIds: Set(members), message: systemMessage)
     }
     
+    /// Loads an array of members for the specified query and updates ``ChatState.members``.
+    ///
+    /// - Note: Channel member sorting keys are set when creating the ``Chat`` instance.
+    /// It is also possible to create separate ``MemberList`` objects if needed with different filtering options. See ``ChatClient.makeMemberList(with:)``.
+    ///
+    /// - Parameter pagination: The pagination configuration which includes a limit and an offset or a cursor.
+    ///
+    /// - Throws: An error while communicating with the Stream API.
+    /// - Returns: An array of paginated channel members for the query.
+    @discardableResult public func loadMembers(with pagination: Pagination) async throws -> [ChatChannelMember] {
+        try await memberList.loadMembers(with: pagination)
+    }
+    
+    // MARK: - Member Moderation
+    
+    /// Bans the specified member from the channel.
+    ///
+    /// When the channel member is banned, they will not be allowed to post messages until the ban is removed
+    /// or expired.
+    ///
+    /// Learn more about [banning and moderation tools](https://getstream.io/chat/docs/ios-swift/moderation/?language=swift#ban).
+    ///
+    ///  - Note: Channel watchers cannot be banned.
+    ///  - Note: In most cases, only admins or moderators are allowed to ban other users from the channel.
+    ///
+    /// - Parameters:
+    ///   - userId: The user id of the channel member.
+    ///   - reason: The reason that the ban was created.
+    ///   - timeoutInMinutes: The number of minutes the user should be banned for. Nil means that the user is banned forever or until the user is unbanned explicitly.
+    ///
+    /// - Throws: An error while communicating with the Stream API or missing required capabilities.
+    public func banMember(_ userId: UserId, reason: String? = nil, timeout timeoutInMinutes: Int? = nil) async throws {
+        try state.channel?.requireCapability(of: .banChannelMembers)
```

Comment:
I'm not sure this should be part of the LLC SDK 🤔 This is more like a UI level flag, and not for LLC, since the backend will respond with a failure if the capability is not there 🤔 This was not in the previous request right?

Reply:
It is in a bit mixed state. Sometime we have been checking for capabilities before. I guess, better not to do it on the client level and let the request to fail. I'll remove it (it was not there for controllers).